packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -93,6 +93,10 @@ export const typographyTokens: { [index: string]: TextStyle } = {
     textDecorationLine: "underline",
   },
 
+  /**
+   * Alignments
+   */
```

Comment:
Love the groupings/comments you've added, makes it easier to follow both reviewing this and for Future Us™ to make sense of what's in here and why